extension/webpack.config.js

```diff
@@ -67,7 +67,7 @@ module.exports = {
     contentScript: path.join(sourcePath, 'ContentScript', 'index.ts'),
     pageContentScript: path.join(sourcePath, 'PageContentScript', 'index.ts'),
     popup: path.join(sourcePath, 'Popup', 'index.tsx'),
-    createTransaction: path.join(sourcePath, 'createTransaction', 'index.tsx'),
+    createTransaction: path.join(sourcePath, 'CreateTransaction', 'index.tsx'),
```

Comment:
TODO Is this fix still needed?

Reply:
Oh yeah I did that in my branch too

Reply:
Doesn't show up on macOS because of the case insensitive filesystem thing

Reply:
Yeah, imagine just me and anyone else on Ubuntu/Linux and likely Windows